lib/mixin/comparable_presentation_search_mixin.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:fluffychat/presentation/model/search/presentation_search.dart';
 
-class ComparablePresentationSearchMixin {
+mixin ComparablePresentationSearchMixin {
```

Comment:
why you did it as a `class` before?

Reply:
because class allows more versatile than mixin, but it's not allow now